Projects/App/Sources/UI/Place/AddPlace/AddPlaceSearchViewController.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  AddPlaceSearchViewController.swift
+//  App
+//
+//  Created by Chanhee Jeong on 2022/10/25.
+//  Copyright (c) 2022 zesty. All rights reserved.
+//
+
+import Combine
+import UIKit
+import DesignSystem
+import SnapKit
+
+final class AddPlaceSearchViewController: UIViewController {
+    
+    // MARK: - Properties
+    private let viewModel = AddPlaceViewModel()
+    private var cancelBag = Set<AnyCancellable>()
+    
+    private var searchResults: [Place] = Place.mockData
+    
+    private lazy var searchingTextFieldView = SearchTextField()
+    private lazy var tableView = UITableView(frame: CGRect.zero, style: .grouped)
+    
+    private lazy var searchButton: UIButton = {
+        let largeConfig = UIImage.SymbolConfiguration(pointSize: 16, weight: .bold, scale: .default)
+        let largeBoldDoc = UIImage(systemName: "magnifyingglass", withConfiguration: largeConfig)
+        $0.setImage(largeBoldDoc, for: .normal)
+        $0.backgroundColor = .black
+        $0.tintColor = .white
+        $0.layer.cornerRadius = 45/2
+        $0.clipsToBounds = true
+        $0.addTarget(self, action: #selector(searchButtonDidTap), for: .touchUpInside)
+        return $0
+    }(UIButton())
+    
+    // MARK: - LifeCycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setNavigationBar()
+        configureUI()
+        createLayout()
+        searchingTextFieldView.textField.delegate = self
+        searchingTextFieldView.textField.becomeFirstResponder()
+    }
+    
+    // MARK: - Function
+    @objc func backButtonDidTap() {
+        self.navigationController?.popViewController(animated: true)
+    }
+    
+    @objc func searchButtonDidTap() {
+//        print("검색버튼이 눌렸어요")
+    }
+    
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        searchingTextFieldView.textField.resignFirstResponder()
+    }
+    
+}
+
+// MARK: - UI Function
+
+extension AddPlaceSearchViewController {
+    
+    private func configureUI() {
+        view.backgroundColor = .white // zestyColor(.backgroundColor)
+        tableView.backgroundColor = .white // zestyColor(.backgroundColor)
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.showsVerticalScrollIndicator = false
+        tableView.estimatedRowHeight = UITableView.automaticDimension
+        tableView.register(SearchResultCell.self, forCellReuseIdentifier: "SearchResultCell")
+        tableView.separatorStyle = .none
```

Comment:
p2;

여기에 리스트 드래그하면 자동으로 키보드 사라지도록 이거 한 줄만 추가해주시면 감사하겠습니다.
```suggestion
        tableView.separatorStyle = .none
        tableView.keyboardDismissMode = .onDrag
```

Reply:
✨ 반영